### PR TITLE
feat(qdrant): pass search_params in find

### PR DESCRIPTION
**Pull request by AnneYang720**
Goals:
This PR is related to issue #666  
Change PR #668 to jina branch.

 - [x] update code
 - [x] add test
 - [x] update documentation

### CODE REVIEW

File: docarray/array/storage/qdrant/find.py [hunk 1]
```diff
@@ -51,15 +44,20 @@ def distance(self) -> 'Distance':
         raise NotImplementedError()
 
     def _find_similar_vectors(
-        self, q: 'QdrantArrayType', limit: int = 10, filter: Optional[Dict] = None
+        self,
+        q: 'QdrantArrayType',
+        limit: int = 10,
+        filter: Optional[Dict] = None,
+        search_params: Optional[Dict] = {},
```
Comment (JoanFM) [inline docarray/array/storage/qdrant/find.py:51]: please set `None` as default and handle properly

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: tests/unit/array/storage/qdrant/test_find.py [hunk 2]
```diff
@@ -22,11 +22,13 @@ def test_success_find_with_added_kwargs(start_storage, monkeypatch):
         )
 
     def _mock_search(*args, **kwargs):
-        assert kwargs['search_params'].hnsw_ef == hnsw_ef
+        if kwargs['search_params']:
```
Comment (JoanFM) [inline tests/unit/array/storage/qdrant/test_find.py:25]: this test would pass, even if the feature did not work

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

File: tests/unit/array/storage/qdrant/test_find.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+from docarray import Document, DocumentArray
+import numpy as np
+
+
+def test_success_find_with_added_kwargs(start_storage, monkeypatch):
```
Comment (JoanFM) [inline tests/unit/array/storage/qdrant/test_find.py:5]: You can parametrize the test with `search` params and expected, so u can test both cases

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

**Review by JoanFM — CHANGES_REQUESTED** [review]
Wait until @NicholasDunham approval

**Review by NicholasDunham — APPROVED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**